torch/_prims/__init__.py

```diff
@@ -154,7 +157,12 @@
     #
     # Tensor Creation Prims
     #
+    "empty",
```

Comment:
This will need a rebase because empty, empty_like, full, and full_like are no longer prims